packages/fast-tooling-react/src/navigation-menu/navigation-menu-item.tsx

```diff
@@ -0,0 +1,200 @@
+import React from "react";
+import { get, uniqueId } from "lodash-es";
+import Foundation, {
+    FoundationProps,
+    HandledProps,
+} from "@microsoft/fast-components-foundation-react";
+import {
+    NavigationMenuItemHandledProps,
+    NavigationMenuItemProps,
+    NavigationMenuItemState,
+    NavigationMenuItemUnhandledProps,
+} from "./navigation-menu-item.props";
+import { MenuItem } from "./navigation-menu.props";
+
+export default class NavigationMenuItem extends Foundation<
+    NavigationMenuItemHandledProps,
+    NavigationMenuItemUnhandledProps,
+    NavigationMenuItemState
+> {
+    public static displayName: string = "NavigationMenuItem";
+
+    public static defaultProps: NavigationMenuItemHandledProps = {
+        expanded: void 0,
+        menuItem: void 0,
+    };
+
+    public static getDerivedStateFromProps(
+        nextProps: NavigationMenuItemProps,
+        prevState: NavigationMenuItemState
+    ): null | Partial<NavigationMenuItemState> {
+        if (
+            nextProps.expanded !== prevState.expanded &&
+            typeof nextProps.expanded === "boolean"
+        ) {
+            return {
+                expanded: nextProps.expanded,
+            };
+        }
+
+        return null;
+    }
+
+    private id: string = uniqueId("navigationMenu");
+
+    constructor(props: NavigationMenuItemProps) {
+        super(props);
+
+        this.state = {
+            expanded: this.props.expanded || false,
+        };
+    }
+
+    public render(): React.ReactNode {
+        return (
+            <div role={"none"} className={this.generateClassNames()}>
```

Comment:
`div` elements are inherently meaningless - do you really need to set `role="none"`?